modules/batcher-mergesort/test/test_batchersort.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2021 Solovev Alexandr
+#include <gtest/gtest.h>
+
+#include <algorithm>
+#include <vector>
+
+#include "include/mergesort.h"
+
+TEST(BatcherSort_Test, Test_Creating_Vector_Exception) {
```

Comment:
```suggestion
TEST(MergeSort_Test, Test_Creating_Vector_Exception) {
```
и так во всех тестах, так как сортировка называется "Сортировка слиянием"

Reply:
исправил на Batcher_Merge_Sort_Test